### PR TITLE
Grammatically updated the 'Mentorship' tab.

### PR DESCRIPTION
**Pull request by RaunakN28**
## Fixes Issue

This PR fixes the following issues:

Small grammatical update to the text in the "Mentorship" tab.

closes #issue-number

## Changes proposed

Here comes all the changes proposed through this PR

## Check List (Check all the boxes which are applicable) 

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Add all the screenshots which support your changes

### CODE REVIEW

File: src/components/home/Mentorship/index.jsx [hunk 1]
```diff
@@ -19,14 +19,14 @@ const Mentorship = ({ background = 'static' }) => {
     >
       <ListItem>
         <span className='highlight higlight-white'>
-          A comprehensive answer to all of your mentoring needs
+          A complete solution to address all of your mentoring requirements.
```
Comment (siddhant-khisty) [inline src/components/home/Mentorship/index.jsx:22]:
```suggestion
          A comprehensive solution to address all of your mentoring requirements.
```

File: src/components/home/Mentorship/index.jsx [hunk 2]
```diff
@@ -19,14 +19,14 @@ const Mentorship = ({ background = 'static' }) => {
...
         </span>
         , including educational materials, communities to join, opportunities,
         and much more.
       </ListItem>
       <ListItem>
         <span className='highlight higlight-white'>
-          Roadmaps to help you stay on track
+          Roadmaps that will guide you towards a successful career,
```
Comment (siddhant-khisty) [inline src/components/home/Mentorship/index.jsx:29]:
```suggestion
          Roadmaps to guide you towards a successful career,
```

**Review by siddhant-khisty — CHANGES_REQUESTED** [review]

**Review by siddhant-khisty — APPROVED** [review]
Lgtm

**Comment by AnuragThePathak** [comment]
@RaunakN28 you'll need to fix the errors.

**Comment by RaunakN28** [comment]
@siddhant-khisty I need some help fixing the errors.

**Comment by AnuragThePathak** [comment]
@RaunakN28 mention what help you need directly.

**Comment by siddhant-khisty** [comment]
@RaunakN28 Looks like a linting error. You can use the prettier extension in VScode and simply run it to fix this.

**Comment by AnuragThePathak** [comment]
@RaunakN28 are you still working?

**Comment by RaunakN28** [comment]
@AnuragThePathak No, since I don't have any knowledge regarding solving these errors.

**Comment by AnuragThePathak** [comment]
@RaunakN28 as far I understand, you simply had to insert a new line after 22nd line in src/components/home/Mentorship/index.jsx

Try doing that and push or simply inform us if you are still confused/unable to do that.

**Comment by RaunakN28** [comment]
@AnuragThePathak Ok. I'll try my best.

**Comment by RaunakN28** [comment]
@AnuragThePathak I guess it worked. Thanks for helping.

**Comment by AnuragThePathak** [comment]
@RaunakN28 great job now it's time to fix the conflicts. If you know how to, do it in your own way.

If not use `git pull upstream main --no-rebase` and git or some editor will open (as per your settings, you need to save it as done in your editor and it should conflict merge process.

**Pull request merged**